ios/Podfile

```diff
@@ -36,8 +36,6 @@ end
 
 post_install do |installer|
   installer.pods_project.targets.each do |target|
-   target.build_configurations.each do |config|
```

Comment:
@CyberWake @rutvik11062000 @sumitra19jha Is this the right thing to do to the the IOS pull request checks to work? It looks like some testing has been removed.